internal/cli/pipeline_inspect.go

```diff
@@ -111,10 +141,80 @@ func (c *PipelineInspectCommand) Run(args []string) int {
 		{
 			Name: "Root Step Name", Value: resp.RootStep,
 		},
-	}, terminal.WithInfoStyle())
+		{
+			Name: "Total Steps", Value: len(resp.Pipeline.Steps),
+		},
+	}
 
-	// TODO(briancain): Use graphviz to build a pipeline graph and display in the terminal?
+	s := uint64(c.flagRunSequence)
+	if s > 0 {
+		run, err := c.project.Client().GetPipelineRun(c.Ctx, &pb.GetPipelineRunRequest{
+			Pipeline: &pb.Ref_Pipeline{
+				Ref: &pb.Ref_Pipeline_Id{
+					Id: &pb.Ref_PipelineId{
+						Id: resp.Pipeline.Id,
+					},
+				},
+			},
+			Sequence: s,
+		})
+		if err != nil {
+			c.ui.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+			return 1
+		}
+		startJob, err := c.project.Client().GetJob(c.Ctx, &pb.GetJobRequest{
+			JobId: run.PipelineRun.Jobs[0].Id,
+		})
+		endJob, err := c.project.Client().GetJob(c.Ctx, &pb.GetJobRequest{
+			JobId: run.PipelineRun.Jobs[len(run.PipelineRun.Jobs)-1].Id,
+		})
+		output = append(output, []terminal.NamedValue{
```

Comment:
I think we can get this from the jobs but we should include some git info here like the SHA and maybe the git message for specific runs (maybe just the first job?)

Reply:
Addressed, but Github didn't pick it up.